src/main/java/duke/Event.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+public class Event extends Task{
```

Comment:
Would it make sense to insert a docstring here to describe what the Event class does?